core/build.gradle.kts

```diff
@@ -0,0 +1,66 @@
+import ApplicationConfig.androidCompileSdk
+import ApplicationConfig.androidMinSdk
+import ApplicationConfig.androidTargetSdk
+
+plugins {
+    kotlin(Plugins.multiplatform)
+    id(Plugins.library)
+}
+
+kotlin {
+    android()
+
+    listOf(
+        iosX64(),
+        iosArm64(),
+        //iosSimulatorArm64() sure all ios dependencies support this target
+    ).forEach {
+        it.binaries.framework {
+            baseName = "core"
```

Comment:
please move
"core"
"main"
"src/androidMain/AndroidManifest.xml"
to buildSrc consts

Reply:
Ok, I will within detekt PR